### PR TITLE
Changes to README to make it more consistent for Prebuilt

### PR DESCRIPTION
**Pull request by adityathakurxd**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

- Changes the README for the `web-sdks` and `roomkit-react`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,62 @@
...
+| `roomkit-web` | `@100mslive/roomkit-web` | This is a web component port of the `HMSPrebuilt` component from the `roomkit-react`. If you are not using React,this can be used as a web component. | [README](./packages/roomkit-web)|
+
+For full documentation, visit [100ms.live/docs](https://www.100ms.live/docs)
+
+<br>
+
+## How to integrate?
+The 100ms SDK gives you everything you need to build scalable, high-quality live video and audio experiences. 
+
+We’re building infrastructure to power **live interactions**. Live interaction refers to anything happening in real-time or near real-time. Examples include video and audio calls (such as Google Meet or Zoom), live streaming (like Twitch or YouTube Live), real-time chat messages (often included as part of the previous two), and collaborative tools like whiteboards and document editors. Additionally, there are offshoots of these live interactions, such as post-call recordings or transcriptions, which can be accessed later.
```
Comment (raviteja83) [inline README.md:23]: I don't think we have document editors. what are we referring to when we say that?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,62 @@
...
+The packages folder contains all the SDK's of 100ms. Here is a brief overview of them:
+| Directory | Package | Description | Link |
+|--|--|--|--|
+| `hms-video-store`  | `@100mslive/hms-video-store` | This package contains the core SDK and the reactive store parts.  | [README](./packages/hms-video-store) |
+| `react-icons`  | `@100mslive/react-icons` | This contains all the icons used in the 100ms prebuilt.  | [README](./packages/react-icons) |
+| `react-sdk` | `@100mslive/react-sdk` | This contains the base React Hooks and some commonly used functionalities as React Hooks. | [README](./packages/react-sdk) |
+| `roomkit-react`  | `@100mslive/roomkit-react`| This contains the React components used in the Prebuilt and the Prebuilt component itself.  | [README](./packages/roomkit-react) |
+| `roomkit-web` | `@100mslive/roomkit-web` | This is a web component port of the `HMSPrebuilt` component from the `roomkit-react`. If you are not using React,this can be used as a web component. | [README](./packages/roomkit-web)|
+
+For full documentation, visit [100ms.live/docs](https://www.100ms.live/docs)
```
Comment (raviteja83) [inline README.md:16]: should we go to web documentation?

File: README.md (hunk 1 above)
Comment (raviteja83) [inline README.md:23]: this paragraph seems like it belongs on the website.

File: README.md (hunk 2 above)
Comment (adityathakurxd) [reply README.md:16]: Sure okay, will replace with the web documentation link.

File: README.md (hunk 1 above)
Comment (adityathakurxd) [reply README.md:23]: Removing this.

File: README.md (hunk 2 above)
Comment (adityathakurxd) [reply README.md:16]: What according to you should this ideally redirect to?

File: README.md (hunk 1 above)
Comment (adityathakurxd) [reply README.md]: It is there on the mental model section: https://www.100ms.live/docs/javascript/v2/quickstart/mental-model
Can remove this if it is too verbose.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,62 @@
...
+1. ## Custom UI
+	-	100ms SDKs are powerful and highly extensible to build and support all custom experiences and UI.
+	-	**Related packages include:** `@100mslive/react-sdk`, `@100mslive/hms-video-store` and `@100mslive/react-icons`.
+	-	Get started with integrating the SDK using the [How to Guide](https://www.100ms.live/docs/javascript/v2/how-to-guides/install-the-sdk/integration). <br>
+
+> Navigate to `react-sdk` for the base React Hooks and some commonly used functionalities by clicking [here](./packages/react-sdk).
+
+2.  ## 100ms Prebuilt 
+	- 100ms Prebuilt is a high-level abstraction that enables you to embed video conferencing and/or live streaming UI—with a few lines of code.
+	- **Related packages include:** `roomkit-react` and `roomkit-web`.
```
Comment (KaustubhKumar05) [inline README.md:36]: + no-code customisation?

File: packages/roomkit-react/README.md [hunk 4]
```diff
@@ -40,42 +43,30 @@ export default App() {
...
+|--|--|
+| [RoomLayoutProvider](src/Prebuilt/provider/roomLayoutProvider/index.tsx) | This is a context that contains the configuration from the dashboard [customiser](dashboard.100ms.live). Whatever changes are made in the dashboard customiser are available the next time you join.|
+|[AppStateContext](src/Prebuilt/AppStateContext.tsx)  | Contains the logic to switch between different screens, for example, Preview to Meeting, Meeting to Leave. These transitions are based on the roomState that is available from the reactive store (`useHMSStore(selectHMSRoomState)`). |
+| [PreviewScreen](src/Prebuilt/components/Preview/PreviewScreen.tsx) | Contains the Preview implementation. Contains the Video tile, video, audio toggles and Virtual background and settings along with the name input.|
+| [ConferenceScreen](src/Prebuilt/components/ConferenceScreen.tsx) | This contains the screen once you finish Preview and enter the meeting. This contains the header and footer and the main content.|
+| [VideoStreamingSection](src/Prebuilt/layouts/VideoStreamingSection.tsx) |This is the component that contains the main video rendering components and a sidebar(Chat, Polls, and Others are shown here)  |
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/README.md:65]: "and Others" can be rephrased. "Interactive features like Chat and Polls are displayed here"

File: packages/roomkit-react/README.md [hunk 5]
```diff
@@ -84,14 +75,20 @@ When [`HMSThemeProvider`](src/Theme/ThemeProvider.tsx) is used at the top level,
...
+
+- Make sure whatever new Component/file you create is in `Typescript`.
+
+- Don't forget to add data-testid for actionables like buttons, menus etc.
 
-Make sure whatever new Component/file you create is in `Typescript`.
+- Setup proper tooling ([ESLint](https://eslint.org/) and [Prettier](https://prettier.io/)) in your editor. 
 
-Don't forget to add data-testid for actionables like buttons, menus etc.
+- `yarn lint` will be run before you push the changes, so whenever a push fails, check if there are any lint errors.
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/README.md:86]: This runs on commit, right @raviteja83?

File: README.md (hunk 3 above)
Comment (adityathakurxd) [reply README.md:36]: Adding that

File: packages/roomkit-react/README.md (hunk 4 above)
Comment (adityathakurxd) [reply packages/roomkit-react/README.md]: Done

File: README.md (hunk 2 above)
Comment (adityathakurxd) [reply README.md:16]: My reasoning for this is - 
- The Custom UI link is already included as a link on the github repository.
Screenshot 2024-02-16 at 4 43 53 PM

---

- With the docs link, users can also go to Prebuilt docs

File: packages/roomkit-react/README.md (hunk 5 above)
Comment (raviteja83) [reply packages/roomkit-react/README.md:86]: it used to be on commit. now it runs on push

File: README.md (hunk 1 above)
Comment (raviteja83) [reply README.md]: does not make sense here imo

Comment (adityathakurxd) [reply README.md]: Removing this

Comment (adityathakurxd) [reply README.md]: Have made the changes, let me know if is good to go.

File: README.md [hunk 6]
```diff
@@ -1,8 +1,60 @@
...
+2.  ## 100ms Prebuilt 
+	- 100ms Prebuilt is a high-level abstraction with no-code customization that enables you to embed video conferencing and/or live streaming UI—with a few lines of code.
+	- **Related packages include:** `roomkit-react` and `roomkit-web`.
+	- Get started with 100ms Prebuilt using the [Prebuilt Quickstart for Web](https://www.100ms.live/docs/javascript/v2/quickstart/prebuilt-quickstart). <br>
+
+> Navigate to `roomkit-react` for the React components used in Prebuilt and the Prebuilt component itself by clicking [here](./packages/roomkit-react).
+
+<hr>
+
+![Banner](https://github.com/adityathakurxd/web-sdks/assets/53579386/ddde5f35-2302-4f91-adb0-f08848e12fba)
```
Comment (raviteja83) [inline README.md:41]: should be from 100mslive/web-sdks

File: examples/prebuilt-react-integration/README.md [hunk 7]
```diff
@@ -1,5 +1,11 @@
-### Running locally
+![Banner](https://github.com/adityathakurxd/web-sdks/assets/53579386/713492fd-425a-48b6-b351-43011902288b)
```
[the remark this answers is not in the document]
Comment (adityathakurxd) [reply examples/prebuilt-react-integration/README.md:1]: This seems to be an auto generated link created when dragging and dropping the file. A way could be to add a new folder, upload the banner asset to it and link to that.

File: README.md (hunk 6 above)
Comment (adityathakurxd) [reply README.md]: Changed it to add a Prebuilt banner and use that instead.

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**